src/open_weather_map/weather.rs

```diff
@@ -32,6 +51,14 @@ pub struct City {
     pub coord: Coord,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+pub struct CityDeserialize {
```

Comment:
maybe City struct can be used instead?

you can make `id` field optional

Reply:
Done in [8e9feff](https://github.com/pxp9/weather_bot_rust/pull/17/commits/8e9feff9b546089bf98bdedf9ba69874ff2cf740)